l10n/en/firefox/faq.ftl

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/faq/
+
+# HTML page title
+page-title = There is a way to protect your privacy. Join { -brand-name-firefox }.
+page-desc = Whether you searched for a fast browser that protects your privacy, this FAQ is here to answer the most pressing { -brand-name-firefox }-related questions.
+
+firefox-faq = { -brand-name-firefox } FAQ
+whether-you-searched-privacy = Whether you searched for a fast browser that protects your privacy, this FAQ is here to answer the most pressing { -brand-name-firefox }-related questions.
```

Comment:
Is it okay that this line is a duplicate of L9:`page-desc`? As long as the localization team doesn't have to localize the string twice, this can be ignored. 

cc/ @peiying2

Reply:
I will consolidate 👍